funding.md

```diff
@@ -36,15 +34,15 @@ To keep Grin open and free from controlling influences, the project will attempt
 
 # Friends of Grin
 
-We believe that any donors, large or small, should be given recognition for their contribution. Any individual donor who wishes to be listed on our [Friends of Grin](friends.md) page will be listed, and any corporate or institutional Donors making a significant contribution to Grin's development will have their logo displayed (if desired).
+We believe that any donors, large or small, should be given recognition for their contribution. Any individual donor who wishes to be listed on our [Friends of Grin](friends) page will be listed, and any corporate or institutional Donors making a significant contribution to Grin's development will have their logo displayed (if desired).
 
 We are also happy to accept recurring sponsorships and acknowledge your support prominently on our Friends of Grin page. However, as per our pledges above, we cannot accept sponsorships or donations predicated on being able to unduly influence Grin's direction.
 
 # Donation Format
 
 As in all things Grin, we want to ensure that the funding process remains as transparent as all other aspects of the project. To this end, Grin will actively seek community funding via:
 
-- [A general fund](general_funding.md), proceeds of which will be allocated by Grin's technical council according to project need
```

Comment:
The .md suffix breaks the link when running locally with `bundle exec jekyll serve`. Adding instructions on how to run locally in https://github.com/mimblewimble/site/pull/39.